tasks/configure_vlans.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- name: "check for required fact - Vlan Name"
+  fail:
+    msg: "missing required fact: Vlan Name"
+  loop: "{{ vlans }}"
+  when: ( vlan.name is not defined )
+  loop_control:
+    loop_var: vlan
+  delegate_to: localhost
+
+- name: "check for required fact - Vlan ID"
+  fail:
+    msg: "missing required fact: Vlan ID"
+  loop: "{{ vlans }}"
+  when: ( vlan.id is not defined )
+  loop_control:
+    loop_var: vlan
+  delegate_to: localhost
```

Comment:
This check can be handled using validate_role_spec task and the expected params for configure_vlan can be added in `meta/configure_vlan_spec.yaml` file

Reply:
Done the required changes and added test case.